### PR TITLE
Extend linter config

### PR DESCRIPTION
**Pull request by Grashalmbeisser**
I added an tox.ini for the viur core project in which u can setup ur pep8 preferences
also a pep8 checker command in the Pipfile "pep8check" which allows u to check only ur diffed files.

It runs with `pipenv run pep8check`

~To add the PR check APP use the following link: https://github.com/OrkoHunter/pep8speaks~

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[pycodestyle]
+show-pep8 = True
```
Comment (sveneberth) [inline tox.ini:2]: Please use 
```ini
show-pep8 = False
```

**Review by sveneberth — CHANGES_REQUESTED** [review]

File: tox.ini (hunk 1 above)
Comment (Grashalmbeisser) [reply tox.ini]: the change was implemented

**Review by sveneberth — APPROVED** [review]

**Review by sveneberth — APPROVED** [review]
Sorry for my latest commits...

**Review by sveneberth — CHANGES_REQUESTED** [review]
This PR no longer makes sense as it is, since pep8speaks has stopped working. We have to find another runner or reduce this PR to the pycodestyle config.

See https://github.com/OrkoHunter/pep8speaks/issues/189#issuecomment-1267106628

**Review by sveneberth — APPROVED** [review]

**Review by phorward — APPROVED** [review]

**Pull request merged**